templates/java-spring/src/main/resources/application.yml

```diff
@@ -0,0 +1,3 @@
+spring:
+  rabbitmq:
```

Comment:
Please, note that some people will use `amqp` but not with `RabbitMQ`. Should it be called `amqp` instead?